elastic/beats
# Add ISO8601 as supported timestamp type

Enhancement

## What does this PR do?

This PR allows the ability to parse Zeek logs that are written with ISO8601 timestamps.

## Why is it important?

ISO8601 timestamps provide a way to view raw log timestamps easily.  By being able to keep these timestamps in the raw logs and have them be parsed correctly when being ingested into the Elastic ecosystem, we get the best of both worlds.

## Checklist


## How to test this PR locally
Redef `LogAscii::json_timestamps = JSON::TS_ISO8601;` for Zeek and have it write logs with ISO8601 timestamps.

The `date` processor will fail for the `ts` field, etc.

Apply `ISO8601` as an additional format option for `ts`, etc in `pipeline.yml`.

Verify log is correctly parsed and ingested into Elasticsearch.
